octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -461,16 +464,19 @@ $(function () {
                     setTimeout(() => {
                         let error401Count = self.error401Count;
                         let triggerUrlCount = self.triggerUrlCount;
+                        let triggerData = self.triggerData;
```

Comment:
- `self._handle_session_expired = function (triggerUrl) {` is missing the new added parameter (`settings.data`)
- `self.triggerData` is never updated (check `self.triggerUrlCount`)